packages/devtools_app/lib/src/code_size/file_import_container.dart

```diff
@@ -92,17 +92,12 @@ class _FileImportContainerState extends State<FileImportContainer> {
           onPressed: () {
             // TODO(peterdjlee): Prompt file picker to choose a file.
```

Comment:
this TODO is now obsolete.

Reply:
I think we'd still want the file picker function to be here since the `fileToBeImported` field will be removed after the file picker is implemented.

Reply:
We will still want a file picker so that a user can pick the snapshot file(s) or analyze-apk.json file(s)

Reply:
makes sense. thanks